front/components/assistant/AssistantBrowser.tsx

```diff
@@ -100,21 +93,12 @@ export function AssistantBrowser({
 
   // check the query string for the tab to show, the query param to look for is called "defaultTab"
```

Comment:
nit: you need to update the comment (defaultTab)

Reply:
You also need to update the code that used it if you rename to selectedTab [here](https://github.com/dust-tt/dust/blob/1c77b404a9cd87a4b75870533b2594a3d0fbcfd8/front/components/assistant_builder/AssistantBuilder.tsx#L336)